README.md

```diff
@@ -1 +1,46 @@
-# microverse-Capstone1
\ No newline at end of file
+# microverse-Capstone1
+This is my capstone project for first month at microverse
+![page-image](./imgs/screenshoot.png)
+
+
+# Built With
+  * HTML, CSS
+  * JS
+  * Bootrap 5
+  * ESlint, Stylelint, And hint
+# Major languages
+  * HTML, CSS
+  * JS
+# Frameworks
+  * Bootrap 5
+
+# Live Demo
+[Live Demo](https://ahmed-alfaouq.github.io/microverse-Capstone1/)
+
+# Getting Started
+ 1) open command line
+ 2) write "git clone https://github.com/Ahmed-Alfaouq/microverse-Capstone1.git"
+ 3) Enjoy
+ 
+ Or
+ 1) Click on code tab
+ 2) Double click on index.html
+ 3) Enjoy
+
+# Authors
+ :bearded_person: **Author 1**
+  - Github: [@Ahmed-Alfarouq](https://github.com/Ahmed-Alfaouq)
+  - LinkedIn: [LinkedIn](https://www.linkedin.com/in/ahmed-omar912ba9199/)
+  - Twitter: [@twitter](https://twitter.com/mediocre23534)
+
+# :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the issues page.
+
+# Show your support
+Give a :star: if you like this project!
+
+# Acknowledgments
+Designed by @Cindy-Shin
```

Comment:
[OPTIONAL]
- Add the correct Acknowledgements that gives appropriate credit to[ Cindy Shin](https://www.behance.net/adagio07) the author of the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).